tmexp/cli.py

```diff
@@ -107,7 +107,7 @@ def add_force_arg(self) -> None:
         )
 
     def add_dataset_arg(self, required: bool) -> None:
-        help = "Name of the dataset created by `preprocess`%s."
+        help = "Name of the output dataset%s."
```

Comment:
Sometimes it's an output (`preprocess`), sometimes it's an input (`create-bow`, `merge`). Maybe something even more generic:
```suggestion
        help = "Name of the dataset%s."
```

Reply:
nice spotting that I'd forgotten :+1: